LayoutTests/ipc/coreipc.js

```diff
@@ -0,0 +1,1189 @@
+const MAX_UINT8 = Math.pow(2, 8)-1;
+const MIN_UINT = 0;
+const MAX_UINT16 = Math.pow(2, 16)-1;
+const MAX_UINT32 = Math.pow(2, 32)-1;
+const MAX_UINT64 = Math.pow(2, 64)-1;
+
+const MAX_INT8 = Math.pow(2, 8-1)-1;
+const MIN_INT8 = -Math.pow(2, 8-1)+1;
+const MAX_INT16 = Math.pow(2, 16-1)-1;
+const MIN_INT16 = -Math.pow(2, 16-1)+1;
+const MAX_INT32 = Math.pow(2, 32-1)-1;
+const MIN_INT32 = -Math.pow(2, 32-1)+1;
+const MAX_INT64 = Math.pow(2, 64-1)-1;
+const MIN_INT64 = -Math.pow(2, 64-1)+1;
+
+
+function deepCopy(object) {
+    return JSON.parse(JSON.stringify(object));
+}
+
+const FUZZ_MARKER = [
+    {type: 'uint64_t', value: 0x0},
+    {type: 'uint8_t', value: 0x46},
+    {type: 'uint8_t', value: 0x55},
+    {type: 'uint8_t', value: 0x5a},
+    {type: 'uint8_t', value: 0x5a},
+    {type: 'uint8_t', value: 0x46},
+    {type: 'uint8_t', value: 0x55},
+    {type: 'uint8_t', value: 0x5a},
+    {type: 'uint8_t', value: 0x5a},
+    {type: 'uint64_t', value: 0x41},
+];
+
+class CoreIPCClass {
```

Comment:
If we want to include this test, it needs to be minimized. This feels like adding an entire fuzzer as a test :)

Reply:
@cdumez : oh, this is the new framework fuzzing team has created. We're expected to keep this file up-to-date with fuzzer team's as an abstraction layer.

Reply:
Ah ok.